### PR TITLE
Updated mime type handling

### PR DESCRIPTION
**Pull request by sibartlett**
Refs #LIBR-718

### CODE REVIEW

File: multi_import/formats.py [hunk 1]
```diff
@@ -99,6 +104,8 @@ def write(self, dataset):
 
 
 class CsvFormat(TabLibFileFormat):
+    mime_type = "text/csv"
```
Comment (emintham) [inline multi_import/formats.py:107]: Should this be `content_type` instead?

Comment (sibartlett) [reply multi_import/formats.py:107]: No, the content_type is currently `application/csv` whereas the mime type is `text/csv`

I still need to look into what the significance of that is.

Comment (sibartlett) [reply multi_import/formats.py]: I did some testing, seems there is little impact if I change content_type from `application/csv` to `text/csv`

I have updated the pull request, to make this change - which has helped to simply things.

File: multi_import/helpers/files.py [hunk 2]
```diff
@@ -25,6 +25,23 @@ def decode_contents(file_contents):
...
+
+    # When Content-Type unspecified, defaults to this.
+    # https://sdelements.atlassian.net/browse/LIBR-355
+    # https://stackoverflow.com/questions/12061030/why-am-i-getting-mime-type-of-csv-file-as-application-octet-stream
+    supported_content_types.append("application/octet-stream")
+
+    if file.content_type not in supported_content_types:
+        allowed_extensions = ", ".join([f.extension.upper() for f in import_file_formats])
+        msg = _(
+
```
Comment (emintham) [inline multi_import/helpers/files.py:40]: there's an extra blank line here

**Comment by emintham** [comment]
code +1 otherwise

**Pull request closed without merging**